src/main/g8/src/it/scala/$package$/domain/Postgres.scala

```diff
@@ -1,54 +1,45 @@
 package $package$.domain
 
 import com.dimafeng.testcontainers.PostgreSQLContainer
+import org.testcontainers.utility.DockerLoggerFactory
 import zio._
 import zio.blocking._
-import slick.interop.zio._
-import com.typesafe.config.Config
-import com.typesafe.config.ConfigFactory
-import scala.jdk.CollectionConverters._
 
 object Postgres {
 
-  type Postgres = Has[PostgreSQLContainer]
-
   final case class TestCredentials(username: String, password: String)
```

Comment:
Is this used somewhere now?

Reply:
You are right, I had to delete that but I forgot.